README.md

```diff
@@ -25,12 +25,49 @@ ray tracer written in pure c++, rendered to ppm file format
 ## Must Know Operations
 > in this section, I'll be going over the **must know mathematical concepts** that one must grasp to *understand the functionality of this ray tracer*.
 ### The Nature of Vectors
+**Three-dimensional (3D) vectors** are mathematical entities used to represent quantities that have both ***magnitude*** and ***direction*** in three-dimensional space. They are fundamental in various fields, including physi
 
 ### Vector Dot Products
+**The dot product**, also known as the scalar product, is an operation that takes two **3D vectors** and produces **a scalar** (single numerical value) as its result. The dot product is used to determine the similarity or alignment of two vectors and provides information about ***the angle*** between them. Here's how to calculate the dot product of two 3D vectors, typically denoted as A and B: 
+
+Given two 3D vectors: 
+
+A = (Ax, Ay, Az) 
+B = (Bx, By, Bz) 
+
+The dot product A · B is computed as: A · B = Ax * Bx + Ay * By + Az * Bz
+
+![Thedotproduct](https://github.com/oskccy/raytracer_cpp/assets/149972489/052775a9-418e-4c57-bd3e-1044d5859d62)
+
 
 ### Vector Cross Products
+**The cross product**, also known as the vector product, is an operation that takes two ***3D vectors*** and produces ***a third vector*** as its result. This new vector is **orthogonal (perpendicular)** to both of the original vectors and provides valuable geometric information about their relationship. Here's how to calculate the cross product of two 3D vectors, typically denoted as A and B: 
+
+Given two 3D vectors: 
+
```

Comment:
make this look like equations in textbooks, with the nice formal font. apply this to all math in the docs